### PR TITLE
Add config reader to `start_dev.sh` & basic config

### PR DESCRIPTION
**Pull request by BanjoFox**
Since the app requires environment variables to bet set, I created a `config.toml` file, and added some code to the `setup_dev.sh` script to parse those entries.  The TOML format was used because that's Rusts preferred syntax. 

Also updated the `setup_dev.sh` script to pull in the `websocket_server` bin.

TODO:
- Maybe move these values to the config?
```
export TRUNK_SERVE_PORT=8081
export ACTIX_PORT=8086
```
- Test with actual/URLs

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+LOGIN_URL = "https://127.0.0.59/login"
```
[the remark this answers is not in the document]
Comment (BanjoFox) [reply config.toml:1]: I -believe- that it is independent.  In fact other testing I have done (not tried docker yet) proved that hard-coded variables has been overriding the local environment.

[the remark this answers is not in the document]
Comment (BanjoFox) [reply config.toml:1]: Converting to draft because this is still a work-in-progress.

**Comment by darioalessandro** [comment]
@BanjoFox how's this PR coming together?

**Comment by BanjoFox** [comment]
Gave everything a once over one more time, and looks okay to me 🤞 
Please feel free to comment.

File: start_dev.sh [hunk 2]
```diff
@@ -2,9 +2,17 @@
 
 # WARNING!! use this script while running without docker.
 
+#- 
+# Pull configuration info from /config.toml
+#
+# Read the values from the config.toml file
+source <(grep = config.toml | sed -E 's/ *= */=/g')
```
Comment (darioalessandro) [inline start_dev.sh:9]: I am probably doing something wrong, but seems like the env vars are not being populated, I recommend running: 

```
source config.toml
```

**Review by darioalessandro — CHANGES_REQUESTED** [review]

File: config.toml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+LOGIN_URL="https://127.0.0.1"
+ACTIX_UI_BACKEND_URL="https://127.0.0.1"
+WEBTRANSPORT_HOST="https://127.0.0.1"
+NATS_URL="nats:4222"
```
[the remark this answers is not in the document]
Comment (BanjoFox) [reply config.toml]: Strange... it was working previously.   Regardless, I have set it to `NATS_URL=nats.zoom.rs` to match what is in the current Dockerfile.  I honestly don't recall where I got that config previously.

File: start_dev.sh (hunk 2 above)
Comment (BanjoFox) [reply start_dev.sh]: Nope... looks like you are correct.  Super frustratingly that command (with grep|sed) works fine If I copy/paste as-is to the command line.  So does simply using `source config.toml`.  I am going to agree with you that the simpler way without sed is better.

**Comment by BanjoFox** [comment]
After making the changes requested, I did re-run the script via VS Code and it was able to bring up the server, and give me a login screen.

**Comment by darioalessandro** [comment]
@BanjoFox can you check if the UI is starting but the backend is crashing due to the failure to connecting to the nats server?

Imo it is ok if it fails as long as we document that people are expected to launch their own nats server locally, you could also look at running `nats-server`

**Pull request closed without merging**